packages/near-fast-auth-signer/src/utils/multi-chain/multiChain.ts

```diff
@@ -1,57 +1,11 @@
+import axios from 'axios';
 import * as bitcoin from 'bitcoinjs-lib';
-import { Account } from 'near-api-js';
 
 import { Bitcoin } from './chains/Bitcoin';
 import EVM from './chains/EVM';
+import { BTCTransaction, Request, Response } from './chains/types';
 
-type BaseTransaction = {
-  to: string;
-  value: string;
-  derivedPath: string
-};
-
-type EVMTransaction = BaseTransaction
-
-type BTCTransaction = BaseTransaction
-
-type BaseChainConfig = {
-  providerUrl: string,
-  scanUrl: string,
-}
-
-type EVMChainConfig = {
-  type: 'EVM',
-}
-
-type BTCChainConfig = {
-  type: 'BTC'
-  networkType: 'bitcoin' | 'testnet'
-}
-
-type ChainConfig = EVMChainConfig | BTCChainConfig
-
-type RequestChainConfig = BaseChainConfig & ChainConfig
-
-type Request = {
-  transaction: EVMTransaction | BTCTransaction;
-  chainConfig: RequestChainConfig;
-  account: Account;
-  fastAuthRelayerUrl: string;
-};
-
-type SuccessResponse = {
-  transactionHash: string;
-  success: true;
-}
-
-type FailureResponse = {
-  success: false;
-  errorMessage: string;
-}
-
-type Response = SuccessResponse | FailureResponse
-
-const MPC_ROOT_PUBLIC_KEY =  'secp256k1:4HFcTSodRLVCGNVcGc4Mf2fwBBBxv9jxkGdiW2S2CA1y6UpVVRWKj6RX7d7TDt65k2Bj3w9FU4BGtt43ZvuhCnNt';
+const MPC_ROOT_PUBLIC_KEY = 'secp256k1:4HFcTSodRLVCGNVcGc4Mf2fwBBBxv9jxkGdiW2S2CA1y6UpVVRWKj6RX7d7TDt65k2Bj3w9FU4BGtt43ZvuhCnNt';
```

Comment:
the mpc contract accountId needs to be passed in as a config and we need to fetch this from there